build.sbt

```diff
@@ -12,21 +11,26 @@ lazy val jvmSdkVersion = "1.1.0"
  */
 
 lazy val `commercetools-sunrise-payment` = (project in file("."))
-  .aggregate(`common`, `payone`)
+  .aggregate(`common`, `payone-adapter`)
   .settings(javaUnidocSettings ++ commonSettings : _*)
+  .settings(
+      libraryDependencies ++= Seq (
+        "com.novocode" % "junit-interface" % "0.11",
+        "org.assertj" % "assertj-core" % "3.4.1"
+      )
+  )
+  .dependsOn(`common`, `payone-adapter`)
 
 lazy val `common` = project
   .configs(IntegrationTest)
   .settings(commonSettings ++ commonTestSettings : _*)
   .settings(
     libraryDependencies ++= Seq(
-      "com.commercetools.sdk.jvm.core" % "commercetools-models" % jvmSdkVersion,
-      "com.commercetools.sdk.jvm.core" % "commercetools-java-client" % jvmSdkVersion,
```

Comment:
Won't you need the client later to be able to push the payments/transactions to the commercetools platform?